includes/Renderer/Stories/Renderer.php

```diff
@@ -624,6 +624,21 @@ protected function render_story_with_poster() {
 		}
 	}
 
+	/**
+	 * Allows customization of html attributes in the web stories widget anchor tag loop
+	 * Converts array into escaped inline html attributes
+	 *
+	 * @return void
+	 */
+	protected function render_attributes() {
```

Comment:
Maybe rename to `render_link_attributes()` to be more specific?

Reply:
Sounds good!